pkg/sql/sem/tree/create.go

```diff
@@ -1703,6 +1703,19 @@ func (node *SequenceOptions) Format(ctx *FmtCtx) {
 			} else {
 				ctx.Printf("%d", *option.IntVal)
 			}
+		case SeqOptRestart:
+			ctx.WriteString(option.Name)
+			ctx.WriteByte(' ')
+			if option.OptionalWord {
+				ctx.WriteString("WITH ")
+			}
+			// TODO(knz): replace all this with ctx.FormatNode if/when
+			// the restart option supports expressions.
+			if ctx.flags.HasFlags(FmtHideConstants) {
+				ctx.WriteByte('0')
```

Comment:
is there a reason we choose `0` instead of `_`?

Reply:
removed `0` since restart should be able to run without value, thx

Reply:
we should still use `_`